_docs/concepts/what-is-istio/overview.md

```diff
@@ -40,17 +45,59 @@ initially focused on Kubernetes but are working to support other environments so
 - **Integration and Customization**. The policy enforcement component can be extended and customized to integrate with existing solutions for 
 ACLs, logging, monitoring, quotas, auditing and more.
 
-These capabilities greatly decrease the coupling between application code, the underlying platform and policy. This decreased coupling not only makes 
-services easier to implement but also makes it simpler for operators to move application deployments between environments or to new policy schemes. 
+These capabilities greatly decrease the coupling between application code, the underlying platform, and policy. This decreased coupling not only makes 
+services easier to implement, but also makes it simpler for operators to move application deployments between environments or to new policy schemes. 
 Applications become inherently more portable as a result.
 
-Istio’s service mesh is logically split into a *data plane* and a *control plane*. The data plane is composed of a set of intelligent (HTTP, HTTP/2, gRPC, TCP or UDP) 
-proxies deployed as sidecars that mediate and control all network communication between microservices. The control plane is responsible for managing and 
-configuring proxies to route traffic, as well as enforce policies at runtime. 
+## Architecture
+
+An Istio service mesh is logically split into a **data plane** and a **control plane**.
+
+- The **data plane** is composed of a set of intelligent
+proxies (Envoy) deployed as sidecars that mediate and control all network communication between microservices.
+- The **control plane** is responsible for managing and 
```

Comment:
Ugh, actually I've just realized if you leave blank lines between bullets you get <p>s inside <li>s and if you don't, you don't, and there's different CSS depending on which one you get.

Personally I actually prefer the non-blank-line version because paragraphs inside list elements is weird HTML, or at least not something I've ever done and I've been writing HTML for years. Let me have a think about that? Also expecting everyone to put blank lines between bulleted lists in Markdown is odd Markdown.

In addition, ARGH MARKDOWN.